pkg/doc/verifiable/common.go

```diff
@@ -3,14 +3,14 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-// Package verifiable provides implementation of Verifiable Credential and Presentation
```

Comment:
You should still have a package comment so that GoDoc index shows up properly.

https://godoc.org/github.com/hyperledger/aries-framework-go/pkg/doc

Reply:
Or can be added in separate doc.go file for package level comments - [ref](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/framework/aries/doc.go).

Reply:
Fixed - thanks guys!